ui/webui/test/helpers/integration.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+from subprocess import CalledProcessError
+
+# import Cockpit's machinery for test VMs and its browser test API
+TEST_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(TEST_DIR, "helpers"))
+sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
+from installer import Installer
+from language import Language
```

Comment:
Ehm, we should eventually fix imports in the Web UI tests. This will very soon start to collide with other Python packages. @VladimirSlavik  @M4rtinK

Reply:
It would be good to have absolute imports. But here they are probably absolute already :-/

Reply:
Related to the discussion below - let's overhaul the way we import these helpers, fixing names can be part of that.